content/work/medcurbside.rst

```diff
@@ -0,0 +1,53 @@
+public: yes
+location: case-studies
+pageclasses: work-layout
+grid_template: true
+image:
+  - src: 'projects/medcurbside.jpg'
+client:
+  - name: 'Lab06'
+project:
+  - name: 'MedCurbside'
+    tagline: 'Ask, Answer, Cite. Make Medicine Better.'
+    url: 'https://www.medcurbside.com/'
+    years: '2016–2017'
+    status: 'live'
+tasks:
+  - Product Branding
+  - User Experience Design
+  - Project Architecture
+  - Responsive Styles
+quotes:
+  - text: |
+      Other developers build exactly what you say,
+      or they don’t understand,
```

Comment:
This is very minor, but I don't like the use of smart-quotes where not needed. It masks the fact that we're converting them anyway, and makes find/replace less consistent. Thoughts?

Reply:
That's fine. I imagine most are leftover from before we started converting all of them.